### ISSUE TITLE
Have more details on tasks list in Dashboard issue

### ISSUE THREAD

**Issue by ademar59** [renovatebot/renovate#17749]
### What would you like Renovate to be able to do?

I want to have the list of dependencies name and version in the task list

You cans see below, an idea of that
I have added the table you have in the MR 

![image](https://user-images.githubusercontent.com/8971839/189618879-6024f14a-3577-4c72-b97e-7ff7b4a24950.png)

### If you have any ideas on how this should be implemented, please tell us here.

Get same code you have in MR description

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by HonkingGoose** [comment]
This reminds me a bit of another issue, that we closed because it was too hard to do:

- https://github.com/renovatebot/renovate/issues/10924

I'll let the maintainers decide if we want to work on this feature. 😉

**Comment by rarkins** [comment]
Mockups of how the markdown would look would be welcome. We're not against the idea, but against doing it "badly" or having a lot of unsatisfied edge cases

**Comment by ademar59** [comment]
Its just by adding the same table like you have in MR (i send you actual and desired md files)
[desired.md](https://github.com/renovatebot/renovate/files/9557196/desired.md)
[actual.md](https://github.com/renovatebot/renovate/files/9557197/actual.md)


| Package | Update | Change |
|---|---|---|
| [datadog](https://www.datadoghq.com) ([source](https://github.com/DataDog/helm-charts)) | patch | `2.37.6` -> `2.37.9` |
| [policy-reporter](https://kyverno.github.io/policy-reporter) ([source](https://github.com/kyverno/policy-reporter)) | minor | `2.11.3` -> `2.12.0` |

**Comment by rarkins** [comment]
Can you copy/past each of those into a comment here? It's important to see how they display in GitHub flavored markdown

**Comment by ademar59** [comment]
I have copied the block in the issue in the  last comment. I dont understand what more i can do

**Comment by ademar59** [comment]
:warning: **Never close this issue, it will disable Renovate permanently**.

This issue lists the debugging tasks to be updated.

It updates periodically and if there are no tasks, it just means you are up to date.

## Pending Approval

These branches will be created by Renovate only once you click their checkbox below.

 - [ ] chore(deps): update helm release datadog to v3

## Open

These updates have all been created already. Click a checkbox below to force a retry/rebase of any.

 - [ ] [chore(deps): update minor](!72) 

| Package | Update | Change |
|---|---|---|
| [datadog](https://www.datadoghq.com) ([source](https://github.com/DataDog/helm-charts)) | patch | `2.37.6` -> `2.37.9` |
| [policy-reporter](https://kyverno.github.io/policy-reporter) ([source](https://github.com/kyverno/policy-reporter)) | minor | `2.11.3` -> `2.12.0` |)


 - [ ] [chore(deps): update minor](!72)

| Package | Update | Change |
|---|---|---|
| [datadog](https://www.datadoghq.com) ([source](https://github.com/DataDog/helm-charts)) | patch | `2.37.6` -> `2.37.9` |
| [policy-reporter](https://kyverno.github.io/policy-reporter) ([source](https://github.com/kyverno/policy-reporter)) | minor | `2.11.3` -> `2.12.0` |

 - [ ] [chore(deps): update minor](!72)

| Package | Update | Change |
|---|---|---|
| [datadog](https://www.datadoghq.com) ([source](https://github.com/DataDog/helm-charts)) | patch | `2.37.6` -> `2.37.9` |
| [policy-reporter](https://kyverno.github.io/policy-reporter) ([source](https://github.com/kyverno/policy-reporter)) | minor | `2.11.3` -> `2.12.0` |


## Detected dependencies

<details><summary>gitlabci</summary>
<blockquote>

<details><summary>.gitlab-ci.yml</summary>


</details>

</blockquote>
</details>

<details><summary>gitlabci-include</summary>
<blockquote>

<details><summary>.gitlab-ci.yml</summary>


</details>

<details><summary>gitlab/deploy.gitlab-ci.yml</summary>


</details>

</blockquote>
</details>

<details><summary>helmv3</summary>
<blockquote>

<details><summary>Chart.yaml</summary>

 - `datadog 2.37.6`
 - `kyverno v2.5.3`
 - `policy-reporter 2.11.3`

</details>

</blockquote>
</details>

---
## Caution
- When you tick a checkbox, you have to wait until new schedule to generate the MR (every 4 hours)
- You can run on demand scan with this <a href="https://gitlab.com/mobivia/driverservices/cloud-platform/digipaas-components/renovate/renovate/-/pipelines/new?var[RENOVATE_ONBOARDING]=false&var[RENOVATE_EXTRA_FLAGS]=mobivia/driverservices/cloud-platform/digipaas-components/digipaas-admin/digipaas-admin-ops" target="_blank">pipeline</a>

**Comment by rarkins** [comment]
<img width="566" alt="image" src="https://user-images.githubusercontent.com/6311784/191665596-9c79d421-5477-47b2-9c0b-3133404d8cb5.png">

I don't think this is true. Closing the issue does not disable Renovate. This seems unrelated to your request for *details* per PR, so would be best if you remove such changes so that we can focus on the one at hand. Please update the comments above

**Comment by ademar59** [comment]
done

**Comment by rarkins** [comment]
1. I think you'd be at risk of exhausting the character limit of github issues
2. I don't think we'll have the possibility of "one size fits all" for a detailed mode - people will have different opinions

So maybe what we could do is have different "templates" rather than changing it permanently or using something like expandedMode true/false.

**Comment by ademar59** [comment]
1. this can be more concise, just so that we can have the version change information (a bit like in the onboarding MR)
2. this can be activated via an option as you propose

**Comment by rarkins** [comment]
So I think we'd need to introduce a new option `dependencyDashboardDisplay` with a default value of `"default"` representing today's approach. And then introducing various other options which can be switched to.

**Comment by Liquidmasl** [comment]
for our mono repo it would be awesome if the dashboard would list _where_ those dependencies are from. Currently a PR has to be created, and then the edited files have to be checked to find out.
